### ISSUE TITLE
Syncplay issues

### ISSUE THREAD

**Issue by exalented** [jellyfin/jellyfin-webos#41]
Please forgive me if this is the wrong place to submit this issue. I have upgraded my server to 10.7.0 (yay!) and syncplay seems to be working properly across other devices, but not on my TV. It seems like the client tell s the server (and by extension the other clients) that it's joined a group, but then doesn't auto-play media nor do any syncing of media. Have others experienced this issue?

**Comment by anthonylavado** [comment]
I don't have an LG TV of my own to test with here, but I will say that SyncPlay support is actually very difficult to get right. We'll have to dig deeper on the jellyfin-web side, but no guarantees for the time being.

**Comment by exalented** [comment]
If it helps any I've found that the exo client shares this same issue across different devices.

**Comment by exalented** [comment]
With SpeedToSync disabled this choppy playback does not occur!

**Comment by nearcatch** [comment]
Has anyone tried SyncPlay recently? I'm running the current 10.8 beta and it failed miserably. Two other people were logging in from desktops, but I tried to join from my webOS app and it just continually added users with my name, without syncing playback at all.

**Comment by exalented** [comment]
Yes, I've tested. It's still unstable it seems. I've also seen the same issue that you explain and it's sorta random I'd say 1/5 times it occurs when you try to join a group.

**Comment by nearcatch** [comment]
> Yes, I've tested. It's still unstable it seems. I've also seen the same issue that you explain and it's sorta random I'd say 1/5 times it occurs when you try to join a group.

I guess I'm unsure if I should be asking here or in jellyfin-web. When I've tried 10.8's SyncPlay using only desktop browsers it's worked great so far. But since jellyfin-webos is just a wrapper for the client, I'm not sure who to report to. Is the current availalbe version of jellyfin-webos wrapped around 10.7? I installed my app from the Homebrew channel.

**Comment by anthonylavado** [comment]
The wrapper loads the web interface from the server you're using.

In general, SyncPlay currently works best with desktop browsers, and not in more places at the moment. The Web/Server teams know. I don't have any further info myself :-(
